src/sql/parts/dashboard/widgets/insights/views/charts/chartInsight.component.ts

```diff
@@ -255,17 +255,17 @@ export abstract class ChartInsight extends Disposable implements IInsightsView {
 			}
 		} else {
 			if (this._config.columnsAsLabels) {
-				return this._data.rows[0].map((row, i) => {
+				return this._data.rows[0].slice(1).map((row, i) => {
```

Comment:
Why if we are using the columns as the labels should we cut off the first column?

Reply:
This config option is really "column names as series labels". Whether it's true or not we use the data in the first column as an axis label, so we shouldn't include it as actual data

Reply:
The problem is running `select 1 as A` and trying to use the chart viewer will cause it to crash. Which is I removed this in the first place. I think fixing the chart insight is a bigger deal than running `select 1 as A` so we should probably merge this in, however, we should reopen that bug and think through this further.

Reply:
`slice` returns an empty array in that case so this block of code behaves fine. It does crash since then there's no data for the chart to display, but that will be handled by #1226 so I don't think there's any more work here

Reply:
The problem is I would expect `select 1 as a` would use a single 1 as a data point for the graphs. Its an edge case, but functionally thats what it should do.